app.py

```diff
@@ -1,12 +1,47 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import pprint
+pp = pprint.PrettyPrinter(indent=4)
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
-def index():
-    return 'Hello World!'
-
+def index(): 
+    items = session.get_items()
+    return render_template('index.html' , items=items)
 if __name__ == '__main__':
```

Comment:
It's slightly confusing to have this `if __name__ == '__main__':` hidden between two route handlers. It should either be straight after app is defined or at the end of the file.
But actually, it's not needed when you start the app with `flask run`. It's only if you try to run the file with python directly. So you can just delete it if you want.